sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientTest.java

```diff
@@ -849,12 +849,15 @@ void canPerformMultipleReceive() {
             .expectNextCount(numberOfEvents)
             .verifyComplete();
 
-        StepVerifier.create(receiver.receiveMessages().take(numberOfEvents))
-            .then(() -> messages.forEach(m -> messageSink.next(m)))
-            .expectNextCount(numberOfEvents)
-            .verifyComplete();
+        // TODO: Do we need to support multiple calls of receiver.receiveMessages()?
+        //  After the autoConnect was removed from ServiceBusAsyncConsumer.processor, the receiver doesn't support
+        //  multiple calls of receiver.receiveMessages().
+//        StepVerifier.create(receiver.receiveMessages().take(numberOfEvents))
+//            .then(() -> messages.forEach(m -> messageSink.next(m)))
+//            .expectNextCount(numberOfEvents)
+//            .verifyComplete();
```

Comment:
We need to support multiple calls to `receiveMessages`. So, this should be fixed.

Reply:
Fixed after I add publish().autoConnect() to receiveMessages() implementation.

Reply:
Discussed with Srikanta again. We agreed to remove the publish and autoConnect and don't support multiple subscribers now.
In the future, we may create a link for each subscriber. To be discussed in the future.